test_pull_requests

```diff
@@ -546,9 +584,60 @@ module Hub
       return pull_request, bot_comment, comments
     end
 
-    def local_merge_pull_request(pull_id, repo)
-      $stderr.puts "Local merging pull request ##{pull_id} for repo '#{repo}'"
+    def get_base_commit_id_for_merge(pull_request, settings, num_tries=5)
+      repo = pull_request['base']['repo']['name']
+      sleep_time = 5
+      base_commit = nil
+      $stderr.puts "Checking if current base repo commit ID matches what we expect"
+      if ENV['BUILD_URL'].to_s.empty?
```

Comment:
It can be string or nil. This is neater than saying `if ENV['BUILD_URL'].nil? || ENV['BUILD_URL'].empty?`